src/res/translation/translation_nl_NL.ts

```diff
@@ -276,7 +276,7 @@
     </message>
     <message>
         <source>Displays the pre-fader audio level of this channel.  All connected clients at the server will be assigned an audio level, the same value for each client.</source>
-        <translation type="vanished">Geeft het pre-fader-audioniveau van dit kanaal weer.  Alle verbonden clients op de server krijgen een audioniveau toegewezen, dezelfde waarde voor elke client.</translation>
+        <translation type="vanished">Geeft het pre-fader-audioniveau van dit kanaal weer. Alle verbonden clients op de server krijgen een audioniveau toegewezen, dezelfde waarde voor elke client.</translation>
```

Comment:
The English has two spaces for a new sentence here (which is fairly normal).  Does the Dutch _consistently_ only have one?  If not, I'd rather things were left alone until a decision is taken on which form to follow.

Reply:
In Dutch, full stops are consistently followed by a single space. Using double spaces would look weird. There was a time (the age of typewriters), that double spaces after full stops were common in Dutch typewriting (like for English and other languages at the time) because a single monospaced typewriter space was not sufficient to show the larger inter-sentence space (the em-quad) that was commonly used in traditional typesetting of books. However, especially in Dutch it mostly disappeared after the dawn of computers. 

Interestingly, in French it is common to have spaces _precede_ exclamation points, question marks, colons, and semi-colons. For instance:

```xml
 187         <source>Personal Mix at: </source>¶
 188         <translation>Mixage personnel à : </translation>¶

2859         <source>Do you want to open the ASIO driver setups?</source>¶
2860         <translation>Voulez-vous ouvrir les configurations des pilotes ASIO ?</translation>¶
```

Reply:
>  English has two spaces for a new sentence here (which is fairly normal)

That never occurred to me! :-) I'd [assumed](https://www.writing-skills.com/one-space-two-full-stop) we were using singles on Jamulus... I'm pretty sure we are?

Reply:
> Interestingly, in French it is common to have spaces precede exclamation points, question marks, colons, and semi-colons. 

Not only "common" but "obligatory" (since given by the French writing rules).

Reply:
> Not only "common" but "obligatory" 

And, being French, the punishments are probably quite severe ;)...

Reply:
Probably something very hard like... an English typographic writing lesson ;)